Recrutify/Recrutify.Host/Configuration/Profiles/ScheduleProfile.cs

```diff
@@ -9,9 +9,10 @@ public ScheduleProfile()
         {
             CreateMap<Schedule, ScheduleDTO>();
             CreateMap<UserPrimarySkill, UserPrimarySkillDTO>();
-            CreateMap<ScheduleSlot, ScheduleSlotDTO>();
-            CreateMap<ScheduleCandidateInfo, ScheduleCandidateInfoDTO>();
-            CreateMap<ScheduleCandidateProjectResult, ScheduleCandidateProjectResultDTO>();
+            CreateMap<ScheduleSlot, ScheduleSlotDTO>().ReverseMap();
```

Comment:
Do you need all these 8 mappings? Do you use them all?

Reply:
yes